app/controllers/hyrax/admin/admin_sets_controller.rb

```diff
@@ -99,11 +99,15 @@ def self.local_prefixes
     private
 
     def update_referer
-      hyrax.edit_admin_admin_set_path(@admin_set) + (params[:referer_anchor] || '')
+      hyrax.edit_admin_admin_set_path(admin_set_id) + (params[:referer_anchor] || '')
     end
 
     def create_admin_set
-      admin_set_create_service.call(admin_set: @admin_set, creating_user: current_user)
+      updated_admin_set = admin_set_create_service.call!(admin_set: admin_set_resource, creating_user: current_user)
+      update_admin_set(updated_admin_set)
+      true
+    rescue RuntimeError
+      false
```

Comment:
Returning Boolean maintains the current behavior based on the `create_admin_set` methods results.

Reply:
if we're going to swallow all errors, let's at least log them?

Reply:
I really don't like swallowing error.  I was doing a minimal change here.  One click up is to log the error.  If we show it to the user, we'd probably have to pretty it up.  Things like `Valkyrie::ObjectNotFound` is an ugly error to show an end user.